lib/mayaUsd/ufe/UsdUndoInsertChildCommand.h

```diff
@@ -64,6 +64,7 @@ class MAYAUSD_CORE_PUBLIC UsdUndoInsertChildCommand : public Ufe::InsertChildCom
     bool insertChildUndo();
 
     UsdSceneItem::Ptr _ufeDstItem;
+    UsdSceneItem::Ptr _ufeParentItem;
```

Comment:
We only seem to use the _ufeParentItem's path, so we should probably store that as the data member.  Whenever mutating operations on the graph get involved (e.g. parenting), and you undo / redo, stale prims (i.e. stale scene items) are always a concern, so using the best practice of storing a Ufe::Path in a command applies here.

Reply:
That is actually what I tried first. It failed the testParentCmd.testParentToProxyShape test because getting back the proxy shape from the parent path requires work. This is why I store the parent Ufe item instead to keep the code simple.